### ISSUE TITLE
Expand all/Collapse all

### ISSUE THREAD

**Issue by simonschwartz** [AusDTO/gov-au-ui-kit#334]
## Description

When a group of accordions is used they should be accompanied by 'Expand all'/'Collapse all' buttons.

![accordions](https://cloud.githubusercontent.com/assets/12635736/17892106/fde1a9ac-6982-11e6-839f-8f738e2af0cf.gif)

The test may also elude to the desired functionality:
https://github.com/AusDTO/gov-au-beta/blob/develop/spec/features/accordion_group.rb#L22-L36
## An MVP approach for reference

https://github.com/AusDTO/gov-au-beta/blob/develop/app/assets/javascripts/application.js

This works for IE9+ using a polyfil for `classList`. I ran out of time before I could explore using `.className +=` which may be a nicer approach for older versions of IE.

When JavaScript is disabled the buttons should not be visible because without JavaScript they are non-functional.
## Timeframes

This functionality currently exists in GOV.AU so there is no rush from a GOV.AU perspective to have this incorporated into the UI Kit

**Comment by klepas** [comment]
Tracked under `DESIGN-205`.

**Comment by klepas** [comment]
If we implement accordions in 2.x we will address this.

If so, we will backport this functionality then into 1.x.

May accordions die in hell. ^_^

**Comment by AndrewArch** [comment]
Call me old-fashioned, but I still reckon accordions have their (limited) place :)

**Comment by klepas** [comment]
I *really* want us to actually test this feature with users.

If it has already been tested I *want* to see the results, at minimum to better understand the feature, and to provide better guidance on when/when not to use it.

**Comment by lreichelt** [comment]
you're so old fashioned @AndrewArch ;)
